.github/workflows/nodejs.yml

```diff
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x]
+        node-version: [14.x]
```

Comment:
I think we're supposed to support 12 still

Reply:
@nfstein I've gotten Brent to update YAAS to use Node 14, and personal YAAS with this branch works.

The reason for the upgrade is that my changes use optional chaining, which is only available on 14+.

Reply:
That makes sense, I was thinking about the skits